src/browser/screen.js

```diff
@@ -434,26 +434,35 @@ function ScreenAdapter(screen_container, bus)
         }
     }
 
-    this.update_cursor_scanline = function(start, end)
+    this.update_cursor_scanline = function(start, end, max)
     {
         if(start & 0x20)
```

Comment:
I'd suggest changing this to

```js
this.update_cursor_scanline = function(start, end, visible)
{
    if(visible)
    {
        cursor_element.style.display = "inline";
        cursor_element.style.height = (end - start) + "px";
        cursor_element.style.marginTop = start + "px";
    }
    else
    {
        cursor_element.style.display = "none";
    }
};
```

and moving the rest of the logic into vga.js